gym_torcs.py

```diff
@@ -272,14 +272,14 @@ def make_observaton(self, raw_obs):
             Observation = col.namedtuple('Observaion', names)
 
             # Get RGB from observation
-            image_rgb = self.obs_vision_to_image_rgb(raw_obs[names[8]])
+            image_rgb = self.obs_vision_to_image_rgb(raw_obs[names['img']])
```

Comment:
please change to 
image_rgb = self.obs_vision_to_image_rgb(raw_obs['img'])

Reply:
> please change to
> image_rgb = self.obs_vision_to_image_rgb(raw_obs['img'])

Hello, how do I get the current image of the agent as the input of the neural network?